code/datums/martial.dm

```diff
@@ -536,31 +528,37 @@
 		playsound(get_turf(A), 'sound/effects/hit_kick.ogg', 50, 1, -1)
 		D.apply_damage(10, BRUTE)
 		D.Weaken(3)
-		add_logs(A, D, "CQC sweeped")
+		add_logs(A, D, "cqc sweeped")
 	return 1
 
-/datum/martial_art/CQC/disarm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+/datum/martial_art/cqc/disarm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
 	add_to_streak("D",D)
 	if(check_streak(A,D))
 		return 1
 	if(prob(65))
-		var/obj/item/I = D.get_active_held_item()
-		if(I)
-			if(D.drop_item())
+		if(!D.stat || !D.weakened || !restraining)
+			var/obj/item/I = D.get_active_held_item()
+			D.visible_message("<span class='warning'>[A] strikes [D]'s jaw with their hand!</span>", \
+								"<span class='userdanger'>[A] strikes your jaw, disorienting you!</span>")
+			playsound(get_turf(D), 'sound/weapons/cqchit1.ogg', 50, 1, -1)
+			if(I)
+				D.drop_item()
 				A.put_in_hands(I)
-		D.visible_message("<span class='danger'>[A] has disarmed [D]!</span>", \
-							"<span class='userdanger'>[A] has disarmed [D]!</span>")
-		playsound(D, 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
+			D.Jitter(2)
+			D.apply_damage(5, BRUTE)
 	else
 		D.visible_message("<span class='danger'>[A] attempted to disarm [D]!</span>", \
 							"<span class='userdanger'>[A] attempted to disarm [D]!</span>")
 		playsound(D, 'sound/weapons/punchmiss.ogg', 25, 1, -1)
 	add_logs(A, D, "disarmed with CQC")
-	if(restrained)
+	if(restraining && A.pulling == D)
 		D.visible_message("<span class='danger'>[A] puts [D] into a chokehold!</span>", \
 							"<span class='userdanger'>[A] puts you into a chokehold!</span>")
 		D.SetSleeping(20)
```

Comment:
@jjrcop actually it's my fault for not recognizing that, dude no it's fine, all on me, i just did it on both of them cause i was a fool like dat it's fine mang you helped alot